### DISCUSSION TITLE
Mac Version

### DISCUSSION THREAD

**Discussion by gbplabs** [cgerchenhp/UE_TAPython_Plugin_Release#33] [General]
Hello, first of all, thank you very much for this invaluable contribution. Just wanted to say that it would be great to have a version for those of us who use Unreal on macOS. Thank you again... Best regards, Pablo from Argentina.

**Comment by cgerchenhp** [comment]
Hello Pablo,

Thank you very much for your invaluable feedback and support! I am pleased to inform you that the macOS version is now available. You can download it from the following link:

[TAPython_5_4_4_macOS_v1_2_3](https://github.com/cgerchenhp/UE_TAPython_Plugin_Release/releases/download/v1.2.3-ue5.4.4/TAPython_5_4_4_macOS_v1_2_3.zip)

Thank you again for your support!

Best regards,

ChenHP
